python/pfs/drp/stella/measureFiberNorms.py

```diff
@@ -200,9 +201,9 @@ def run(self, armSpectra: Dict[int, List[PfsArm]], pfsConfig: PfsConfig) -> Stru
         coadded = {spec: self.coaddSpectra(pfsArmList) for spec, pfsArmList in armSpectra.items()}
         fiberNorms = self.measureFiberNorms(coadded, visitSet)
         if self.config.doPlot:
-            plot = self.plotFiberNorms(
-                fiberNorms, pfsConfig, visitSet, next(iter(coadded.values())).identity.arm
-            )
+            arm = next(iter(coadded.values())).identity.arm
+            plot = plotFiberNorms(fiberNorms, pfsConfig, lower=self.config.plotLower, upper=self.config.plotUpper)
+            plot.axes[0].set_title(f"Fiber normalization for arm={arm}\nvisits: {','.join(map(str, visitSet))}")
```

Comment:
I thought the idea was to completely remove the act of plotting from `measureFiberNorms.py` and put it in drp_qa.